keps/sig-windows/4802-windows-node-shutdown/README.md

```diff
@@ -0,0 +1,541 @@
+      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
+      - yes (will require restart of kubelet)
+
+* **Does enabling the feature change any default behavior?**
+  Any change of default behavior may be surprising to users or break existing
+  automations, so be extremely careful here.
+
+    * The main behavior change is that during a node shutdown, pods running on
+      the node will be terminated gracefully.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?**
+  Describe the consequences on existing workloads (e.g., if this is a runtime
+  feature, can it break the existing applications?).
+
+    * Yes, the feature can be disabled by either disabling the feature gate, or
+      setting `kubeletConfig.ShutdownGracePeriod` to 0 seconds.
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+
+    * Kubelet will attempt to perform graceful termination of pods during a
+        node shutdown.
+
+* **Are there any tests for feature enablement/disablement?**
+  The e2e framework does not currently support enabling or disabling feature
+  gates. However, unit tests in each component dealing with managing data, created
+  with and without the feature, are necessary. At the very least, think about
+  conversion tests if API types are being modified.
+
+    *   n/a
+
+### Rollout, Upgrade and Rollback Planning
+
+_This section must be completed when targeting beta graduation to a release._
+
+* **How can a rollout fail? Can it impact already running workloads?**
+  Try to be as paranoid as possible - e.g., what if some components will restart
+   mid-rollout?
+
+This feature should not impact rollouts.
+
+* **What specific metrics should inform a rollback?**
+
+N/A.
+
+* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+  Describe manual testing that was done and the outcomes.
+  Longer term, we may want to require automated upgrade/rollback tests, but we
+  are missing a bunch of machinery and tooling and can't do that now.
+
+The feature is part of kubelet config so updating kubelet config should
+enable/disable the feature; upgrade/downgrade is N/A.
+
+* **Is the rollout accompanied by any deprecations and/or removals of features, APIs,
+fields of API types, flags, etc.?**
+  Even if applying deprecation policies, they may still surprise some users.
+
+No.
+
+### Monitoring Requirements
+
+_This section must be completed when targeting beta graduation to a release._
+
+* **How can an operator determine if the feature is in use by workloads?**
+  Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
+  checking if there are objects with field X set) may be a last resort. Avoid
+  logs or events for this purpose.
+
+Check if the feature gate and kubelet config settings are enabled on a node.
+
+* **What are the SLIs (Service Level Indicators) an operator can use to determine
```

Comment:
`How can someone using this feature know that it is working for their instance?` is missing from the KEP template.

Reply:
Added